api-authorization/server/authorization-middleware.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+const errorMiddleware = require('./error-middleware');
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+  const xAccessToken = req.headers['x-access-token'];
+  if (!xAccessToken) {
+    const error401 = new ClientError(401, 'authentication required');
+    throw errorMiddleware(error401, req, res, next);
```

Comment:
no, i meant throw the error object, your middleware will already get the error because it's at the bottom of `index.js` 😂